pkg/nfd-master/nfd-master.go

```diff
@@ -1161,6 +1169,18 @@ func (m *nfdMaster) configure(filepath string, overrides string) error {
 	if m.args.Overrides.ResyncPeriod != nil {
 		c.ResyncPeriod = *m.args.Overrides.ResyncPeriod
 	}
+	if m.args.Overrides.NfdApiParallelism != nil {
+		c.NfdApiParallelism = *m.args.Overrides.NfdApiParallelism
+	} else {
+		// the default value of the NfdApiParallelism should not be 0
+		if c.NfdApiParallelism == 0 {
+			c.NfdApiParallelism = DefaulMaxNumberOfConcurrentLabelers
+		}
+	}
+
+	if c.NfdApiParallelism < 0 {
+		return fmt.Errorf("the maximum number of concurrent labelers should be a positive number")
+	}
```

Comment:
I think we could simplify and remove the (wrongly named) const alltogether by just adding the default (10) to `newDefaultConfig()` and then banning zero.

```suggestion
	}
	if c.NfdApiParallelism <= 0 {
		return fmt.Errorf("the maximum number of concurrent labelers should be a non-zero positive number")
	}
```

WDYT?

Reply:
Yes that's cleaner.